### ISSUE TITLE
golemcli account info fails with openssl unknown bio failure

### ISSUE THREAD

**Issue by etam** [golemfactory/clay#2569]
```
(venv)  ./golemcli.py account info
Unhandled Error
Traceback (most recent call last):
  File "/home/etam/code/github/golemfactory/golem/venv/lib/python3.6/site-packages/twisted/python/log.py", line 103, in callWithLogger                                                                                                                
    return callWithContext({"system": lp}, func, *args, **kw)
  File "/home/etam/code/github/golemfactory/golem/venv/lib/python3.6/site-packages/twisted/python/log.py", line 86, in callWithContext                                                                                                                
    return context.call({ILogContext: newCtx}, func, *args, **kw)
  File "/home/etam/code/github/golemfactory/golem/venv/lib/python3.6/site-packages/twisted/python/context.py", line 122, in callWithContext                                                                                                           
    return self.currentContext().callWithContext(ctx, func, *args, **kw)
  File "/home/etam/code/github/golemfactory/golem/venv/lib/python3.6/site-packages/twisted/python/context.py", line 85, in callWithContext                                                                                                            
    return func(*args,**kw)
--- <exception caught here> ---
  File "/home/etam/code/github/golemfactory/golem/venv/lib/python3.6/site-packages/twisted/internet/posixbase.py", line 614, in _doReadOrWrite                                                                                                        
    why = selectable.doRead()
  File "/home/etam/code/github/golemfactory/golem/venv/lib/python3.6/site-packages/twisted/internet/tcp.py", line 205, in doRead                                                                                                                      
    return self._dataReceived(data)
  File "/home/etam/code/github/golemfactory/golem/venv/lib/python3.6/site-packages/twisted/internet/tcp.py", line 211, in _dataReceived                                                                                                               
    rval = self.protocol.dataReceived(data)
  File "/home/etam/code/github/golemfactory/golem/venv/lib/python3.6/site-packages/twisted/protocols/tls.py", line 330, in dataReceived                                                                                                               
    self._flushReceiveBIO()
  File "/home/etam/code/github/golemfactory/golem/venv/lib/python3.6/site-packages/twisted/protocols/tls.py", line 300, in _flushReceiveBIO                                                                                                           
    self._flushSendBIO()
  File "/home/etam/code/github/golemfactory/golem/venv/lib/python3.6/site-packages/twisted/protocols/tls.py", line 252, in _flushSendBIO                                                                                                              
    bytes = self._tlsConnection.bio_read(2 ** 15)
  File "/home/etam/code/github/golemfactory/golem/venv/lib/python3.6/site-packages/OpenSSL/SSL.py", line 1672, in bio_read
    self._handle_bio_errors(self._from_ssl, result)
  File "/home/etam/code/github/golemfactory/golem/venv/lib/python3.6/site-packages/OpenSSL/SSL.py", line 1650, in _handle_bio_errors
    raise ValueError("unknown bio failure")
builtins.ValueError: unknown bio failure

RPC: call error: [Failure instance: Traceback (failure with no frames): <class 'autobahn.wamp.exception.ApplicationError'>: ApplicationError(error=<wamp.close.transport_lost>, args=['WAMP transport was lost without closing the session before'], kwargs={}, enc_algo=None)]
[11:30:42.007717] ERROR: Exception: ApplicationError(error=<wamp.close.transport_lost>, args=['WAMP transport was lost without closing the session before'], kwargs={}, enc_algo=None)
```

branch 0.15.0 (2d9c3f95b)

**Comment by etam** [comment]
And now, when trying it again, it prints
```
[11:38:25.149604] ERROR: Exception: [Errno 17] File exists
```

**Comment by etam** [comment]
Fixed in 0.15.0
